src/Jackett.Server/Controllers/ResultsController.cs

```diff
@@ -440,7 +454,18 @@ public async Task<IActionResult> Torznab([FromQuery] TorznabRequest request)
         }
 
         [Route("[action]/{ignored?}")]
-        public IActionResult GetErrorXML(int code, string description) => Content(CreateErrorXML(code, description), "application/xml", Encoding.UTF8);
+        public IActionResult GetErrorXML(int code, string description, int statusCode = StatusCodes.Status400BadRequest)
```

Comment:
should the default be moved to be within the code; looks like that would possibly be more consistent?